kafka/modules/kafka/install.sh

```diff
@@ -2,9 +2,39 @@
 
 set -e
 
+SOURCES_DIR=/tmp/artifacts
 SCRIPT_DIR=$(dirname $0)
 SCRIPTS_DIR=${SCRIPT_DIR}/scripts
+STUNNEL_DIR=${SCRIPT_DIR}/stunnel-scripts
+S2I_DIR=${SCRIPT_DIR}/s2i-scripts
+LICENSE_DIR=/root/licenses
+PRODUCT_LICENSE_DIR=${LICENSE_DIR}/${COM_REDHAT_COMPONENT}
+
+mkdir $KAFKA_HOME
+mkdir $STUNNEL_HOME
+mkdir $S2I_HOME
+mkdir -p -m g+rw /usr/local/var/run/
```

Comment:
from where this line comes out?

Reply:
Its a read/writable directory for stunnel sockets: https://github.com/strimzi/strimzi-kafka-operator/blob/master/docker-images/kafka/Dockerfile#L52